website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -1,32 +1,58 @@
 ---
 layout: docs
-page_title: Service Mesh - What is Cluster Peering?
+page_title: Cluster Peering Overview
 description: >-
-  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn about the cluster peering process, differences with WAN federation for multi-datacenter deployments, and technical constraints.  
+  Cluster peering establishes communication between independent clusters in Consul, allowing services to interact across datacenters. Learn how cluster peering works, its differences with WAN federation for multi-datacenter deployments, and how to troubleshoot common issues.  
 ---
 
-# What is Cluster Peering?
+# Cluster peering overview
 
-You can create peering connections between two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
+This topic provides an overview of cluster peering, which is a process that connects two or more independent clusters so that services deployed to different partitions or datacenters can communicate.
 
-## Overview
+Cluster peering is enabled in Consul by default. For specific information about cluster peering configuration and usage, refer to following pages.
 
-Cluster peering is a process that allows Consul clusters to communicate with each other. The cluster peering process consists of the following steps:
+**Reference information:**
```

Comment:
I would actually change the order of this to the following. this would be more in line with the user journey (beginning to advanced)

```
Tutorials > Usage (runtime specific Kubernetes) > Reference 
```

also, i would combine the kubernetes with the usage one (bold the Kubernetes). this makes it clear that the kubernetes content is in usage

Reply:
on a second read, i think Usage and Reference might be more useful after the background?